### ISSUE TITLE
Anchors disappearing from list

### ISSUE THREAD

**Issue by Aurescalia** [Multiverse/Multiverse-Portals#465]
Hi

 

i have an issues with anchors,

everytime i reboot my serv, the anchors are "invalid position" and all my portals are then broken (since the destination are these anchors)

 

[10:27:06 WARN]: [Multiverse-Core] The location for anchor 'platform' is INVALID.
[10:27:06 WARN]: [Multiverse-Core] The location for anchor 'spawn' is INVALID.
[10:27:06 WARN]: [Multiverse-Core] The location for anchor 'survival' is INVALID.

and then i get this on my portals

[10:27:06 WARN]: [Multiverse-Core] Portal from_creative_to_spawn has an invalid DESTINATION!

  
after investigating a bit, when i used the command "/mv reload" all my anchors disappeared from the list (/mv anchors)
this should be a clue

i have all latest versions of bukkit, mv core and mv portals

**Comment by Jagenka** [comment]
I had the same Problem and fixed it by changing the anchors.yml file.
Changed from:
```
anchors:
  survival_portal: wanderingisles:528,00,51,00,-348,50:180,00:0,00
```
to:
```
anchors:
  survival_portal: wanderingisles:528.00,51.00,-348.50:180.00:0.00
```
I replaced `,` (German decimal point) with `.`

**Comment by blubberdiblub** [comment]
@Jagenka 

Note that that's not a fix, but just a workaround. It will break again any time anchors change from within the game (i.e. anchors are added or removed).

Also, it's a problem with Multiverse-Core, not Multiverse-Portals, as the former manages anchors, the latter just calls up their coordinates/world. See https://github.com/Multiverse/Multiverse-Core/issues/2102

You can work around it by fiddling with your locale settings, but of course that's not satisfactory for server owners in affected locales, to say the least, and not a fix.

I've got a pull request pending here: https://github.com/Multiverse/Multiverse-Core/pull/2103

**Comment by Jagenka** [comment]
@blubberdiblub 
Saw that too afterwards, sorry. I know it's not a fix, but wanted to share nevertheless despite my limited understanding of the code.
I will avoid Anchors for now, as they don't exactly fit my need.

**Comment by nicegamer7** [comment]
Closed by Multiverse/Multiverse-Core#2103.
